### ISSUE TITLE
Debounce not working when passing `requestOptions`

### ISSUE THREAD

**Issue by giopetris** [wellyshen/use-places-autocomplete#116]
# Bug Report

## Describe the Bug

Debounce not working when using `requestOptions`

## How to Reproduce

1. Pass `requestOptions` in the hook:

```
usePlacesAutocomplete({
    requestOptions: {},
    debounce: 300
});
```

## Expected Behavior

But it works when there's no `requestOptions`:

```
usePlacesAutocomplete({
    debounce: 300
});
```

**Comment by wellyshen** [comment]
@giopetris Thank you for reporting this bug. I will fix it ASAP.

**Comment by giopetris** [comment]
Thanks @wellyshen

**Comment by wellyshen** [comment]
@giopetris Fixed via [v1.0.7](https://github.com/wellyshen/use-places-autocomplete/releases/tag/v1.0.7)

**Comment by giopetris** [comment]
Wow, so fast @wellyshen ! Thanks, it's working now
